plugins/account_history/include/golos/plugins/account_history/history_object.hpp

```diff
@@ -22,13 +22,102 @@ namespace plugins {
+
+
+    class operation_object
+            : public object<operation_object_type, operation_object> {
+    public:
+        operation_object() = delete;
+
+        template<typename Constructor, typename Allocator>
+        operation_object(Constructor &&c, allocator <Allocator> a)
+                :serialized_op(a.get_segment_manager()) {
+            c(*this);
+        }
+
+        id_type id;
+
+        transaction_id_type trx_id;
+        uint32_t block = 0;
+        uint32_t trx_in_block = 0;
+        uint16_t op_in_trx = 0;
+        uint64_t virtual_op = 0;
+        time_point_sec timestamp;
+        buffer_type serialized_op;
+    };
+
+    using operation_id_type = object_id<operation_object>;
+
+    struct by_location;
+    struct by_transaction_id;
+    using operation_index = multi_index_container <
+        operation_object,
+            indexed_by<
+                ordered_unique <
+                    tag < by_id >,
+                    member <
+                        operation_object,
+                        operation_id_type,
+                        &operation_object::id
+                        >
+                    >,
+                ordered_unique <
+                    tag < by_location >,
+                    composite_key<
+                        operation_object,
+                        member <
+                            operation_object,
+                            uint32_t,
+                            &operation_object::block
+                        >,
+                        member<
+                            operation_object,
+                            uint32_t,
+                            &operation_object::trx_in_block
+                        >,
+                        member<
+                            operation_object,
+                            uint16_t,
+                            &operation_object::op_in_trx
+                        >,
+                        member<
+                            operation_object,
+                            uint64_t,
+                            &operation_object::virtual_op
+                        >,
+                        member<
+                            operation_object,
+                            operation_id_type,
+                            &operation_object::id
+                        >
+                    >
+                >,
+                ordered_unique <
+                    tag <by_transaction_id>,
+                    composite_key<
+                        operation_object,
+                        member <
+                            operation_object,
+                            transaction_id_type,
+                            &operation_object::trx_id
+                        >,
+                        member <
+                            operation_object,
+                            operation_id_type,
+                            &operation_object::id
+                        >
```

Comment:
In one line to the end of the previous line: `member<operation_object, operation_id_type, &operation_object::id>>>>,`

Reply:
Fixed